Traceback (most recent call last):
  File "/home/heewonyu/Project/Project/Twitter_Crawler/send_mail.py", line 11, in <module>
    DB_HOST = config.get('postgres', 'host')
  File "/usr/lib/python3.10/configparser.py", line 782, in get
    d = self._unify_values(section, vars)
  File "/usr/lib/python3.10/configparser.py", line 1153, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'postgres'
Traceback (most recent call last):
  File "/home/heewonyu/Project/Project/Twitter_Crawler/send_mail.py", line 44, in <module>
    cur.execute(query % year_month_day)
psycopg2.errors.UndefinedTable: 오류:  "public.twt_jg_20230416" 이름의 릴레이션(relation)이 없습니다
LINE 1: SELECT * FROM public."twt_jg_20230416" WHERE created_at >= n...
                      ^

